### DISCUSSION TITLE
Add "Save As" in file menu.

### DISCUSSION THREAD

**Discussion by Meelperd** [phcode-dev/phoenix#1653] [Ideas]
I think the title is self-explanitory?

**Comment by abose** [comment]
Thanks for reporting the issue, its not available in the browser as our virtual fs layer can only mount folders from the os, so mounting files may take a bit longer to implement as the underlying fs layer needs to be updated. 

The native builds supports this feature though.

**Reply by Meelperd** [reply]
Thank you Arun.
I'm not using the browser version. Do I have the correct native version?
Release 3.7 build 3.7.3-20322
build timestamp: 2024-05-25T13:23:21.725Z
Screenshot:
![phcode save as](https://github.com/phcode-dev/phoenix/assets/95029064/3c4722c2-4901-4ad3-a4dc-e6197847b58c)

Great editor, by the way!
Richard

**Reply by abose** [reply]
Oops, looks like i forgot to reenable the option in desktop builds after disabling it in the browser, will fix in june release.

**Reply by abose** [reply]
https://github.com/phcode-dev/phoenix/pull/1656
